editoast/src/views/timetable.rs

```diff
@@ -114,7 +114,7 @@ struct TimetableIdParam {
 
 /// Return a specific timetable with its associated schedules
 #[utoipa::path(
-    get, path = "",
+    get, path = "/train_schedule",
```

Comment:
```suggestion
    get, path = "/train_schedules",
```

we mostly pluralize paths (not consistent though)